### PR TITLE
canal_lock, add intromessage

### PR DESCRIPTION
**Pull request by AkshayCHD**
Add an intro message to canal lock activity to help children figure out how to work with the activity.
screen3

screen2

### CODE REVIEW

File: src/activities/canal_lock/CanalLock.qml [hunk 1]
```diff
@@ -46,6 +46,32 @@ ActivityBase {
...
+                left: parent.left
+                leftMargin: 5
+            }
+            z: 100
+            onIntroDone: {
+                anim.running = true
+                info.visible = true
+            }
+            intro: [
+                qsTr("Your goal is to get Tux across the canal lock to his get wooden logs, "
```
Comment (divyam3897) [inline src/activities/canal_lock/CanalLock.qml]: to get his

File: src/activities/canal_lock/CanalLock.qml [hunk 2]
```diff
@@ -46,6 +46,32 @@ ActivityBase {
...
+                     +" using the different types of water locks avaliable."),
```
Comment (divyam3897) [inline src/activities/canal_lock/CanalLock.qml]: two spaces after "," omit one

File: src/activities/canal_lock/CanalLock.qml [hunk 3]
```diff
@@ -46,6 +46,32 @@ ActivityBase {
...
+                qsTr("The vertical coloured bars represent the water locks, and can be operated by clicking them. "
+                     +"Two locks of the same type cannot be operated simultaneously ") ,
+                qsTr("The water level inside the lock will change according to the side of the canal it is "
+                     +"connected with. Use this property to help tux get his job done.")
```
Comment (divyam3897) [inline src/activities/canal_lock/CanalLock.qml]: Can the sentences be made better? They can even be combined into one.

**Comment by AkshayCHD** [comment]
@petitlapin Please have a look

File: src/activities/canal_lock/CanalLock.qml (hunk 1 above)
Comment (sagaragarwal94) [reply src/activities/canal_lock/CanalLock.qml]: @AkshayCHD @divyam3897 Let it be `to get` not `to get his`/ `to get her`. Do normalize like `to get`

File: src/activities/canal_lock/CanalLock.qml [hunk 4]
```diff
@@ -46,6 +46,27 @@ ActivityBase {
...
+                rightMargin: 5
+                left: parent.left
+                leftMargin: 5
+            }
+            z: 100
+            intro: [      
+                qsTr("Your goal is to get Tux across the canal lock to get his wooden logs, "
+                     +"using the different types of water locks available."),
+                qsTr("The vertical colored bars represent the water locks, which can be operated by clicking them. "
+                     +"Two locks of the same type cannot be operated simultaneously ") ,
```
Comment (petitlapin) [inline src/activities/canal_lock/CanalLock.qml]: missing . at the end

**Comment by AkshayCHD** [comment]
@petitlapin I have made the changes, please review

**Comment by petitlapin** [comment]
Hi,
I've merged it in https://commits.kde.org/gcompris/5b870e82f67f9609af4afb8b5157a5bac521ec7b
I've just changed before commiting:
"Use this property to help Tux get his job done." to "Use this property to help Tux get the job done."
Thank you!

**Pull request closed without merging**